spacy_llm/backends/integration/langchain.py

```diff
@@ -21,22 +21,28 @@ def _check_installation() -> None:
 
 
 @registry.llm_queries("spacy.CallLangChain.v1")
-def query_langchain() -> Callable[["BaseLLM", Iterable[Any]], Iterable[Any]]:
+def query_langchain() -> Callable[
+    ["langchain.llms.base.BaseLLM", Iterable[Any]], Iterable[Any]
```

Comment:
Don't you need to update the `TYPE_CHECKING` block to correspond to this?

Reply:
I don't think it's necessary at all anymore now?